pmm-app/src/pmm-qan/panel/components/Details/Example/Example.tools.tsx

```diff
@@ -1,17 +1,17 @@
-import Highlight from 'react-highlight.js';
 import React from 'react';
 import sqlFormatter from 'sql-formatter';
 import { ReactJSON } from 'shared/components/Elements/ReactJSON/ReactJSON';
 import { Databases } from '../Details.types';
+import { HighlightWrapper } from '../../../../../shared/components/Elements/HighlightWrapper/HighlightWrapper';
```

Comment:
from 'shared/...'

Reply:
Actually, it's premature to put everything in shared. This component should live in /pmm-qan only.

Reply:
Good point, moved back to the QAN panel